src/order/symm_diff.lean

```diff
@@ -75,6 +78,13 @@ end
 
 lemma symm_diff_le_sup : a Δ b ≤ a ⊔ b := by { rw symm_diff_eq_sup_sdiff_inf, exact sdiff_le }
 
+lemma inf_symm_diff_distrib_left : a ⊓ b Δ c = (a ⊓ b) Δ (a ⊓ c) :=
+by rw [symm_diff_eq_sup_sdiff_inf, inf_sdiff_distrib_left, inf_sup_left, inf_inf_distrib_left,
+  symm_diff_eq_sup_sdiff_inf]
+
+lemma inf_symm_diff_distrib_right : a Δ b ⊓ c = (a ⊓ c) Δ (b ⊓ c) :=
```

Comment:
For clarity:
``suggestion
lemma inf_symm_diff_distrib_right : (a Δ b) ⊓ c = (a ⊓ c) Δ (b ⊓ c) :=
```